### ISSUE TITLE
Incorrect derivative of function w.r.t. parameterized argument alpha

### ISSUE THREAD

**Issue by bentyeh** [psturmfels/VisualizingExpectedGradients#5]
This footnote is not quite right.

https://github.com/distillpub/post--attribution-baselines/blob/765dadd4c57418962cdf6056094121f630569645/public/index.html#L267-L278

`$$\gamma(\alpha)$$` is a vector, so by the multivariable chain rule,

```
$$
\frac{\delta f(\gamma(\alpha))}{\delta \alpha}
= \sum_i \frac{\delta f(\gamma(\alpha))}{\delta \gamma_i} \times  
  \frac{\delta \gamma_i(\alpha)}{\delta \alpha}
= \sum_i \frac{\delta f(x' + \alpha' (x - x'))}{\delta x_i} \times (x_i - x'_i)
$$
```

**Comment by psturmfels** [comment]
You are right! That footnote definitely contains errors, and your version is correct. I'm happy to change this myself, but also happy to accept a pull request. Either way, I'll try and update it this evening. Thank you for spotting this!

**Comment by bentyeh** [comment]
I created a pull request: #6
